lib/crowbar/client/command/node/list.rb

```diff
@@ -73,6 +73,10 @@ def content_from(request)
                 )
```

Comment:
But it doesn't, e.g. https://github.com/crowbar/crowbar-client/blob/master/lib/crowbar/client.rb#L21 you didn't write

    Bundler.setup(
      :default
    )

and https://github.com/crowbar/crowbar-client/blob/master/lib/crowbar/client.rb#L60 you didn't write

    super(
      "Barclamp #{barclamp} is not available"
    )

There are many other examples, so I am wondering what is the exact style rule you are talking about and what is the difference between these examples.